service-registry/src/main/java/org/apache/servicecomb/serviceregistry/config/ServiceRegistryConfig.java

```diff
@@ -109,22 +109,23 @@ public HttpVersion getHttpVersion() {
     return HttpVersion.valueOf(property.get());
   }
 
-  public int getWorkerPoolSize() {
-    String workerPoolSizeKey = "servicecomb.service.registry.client.workerPoolSize";
+  public int getInstances() {
+    String instances = "servicecomb.service.registry.client.instances ";
     DynamicIntProperty property =
-        DynamicPropertyFactory.getInstance()
-            .getIntProperty(workerPoolSizeKey, 1);
-    int workerPoolSize = property.get();
-    if (workerPoolSize <= 0) {
+            DynamicPropertyFactory.getInstance()
+                    .getIntProperty(instances, 1);
+    int deployInstances = property.get();
+    if (deployInstances <= 0) {
       int nAvailableProcessors = Runtime.getRuntime().availableProcessors();
       LOGGER.warn("The property `{}` must be positive integer, fallback to use number of available processors: {}",
-          workerPoolSizeKey,
-          nAvailableProcessors);
+              instances,
+              nAvailableProcessors);
```

Comment:
The indent of some files seems not consistent with the other code, like here and `AbstractClientPool`. Could you reformat your pull request? You can find the code template [here](https://github.com/apache/servicecomb-java-chassis/tree/master/etc).